backend/src/main/java/gov/cdc/usds/simplereport/api/accountrequest/IdentityVerificationController.java

```diff
@@ -144,6 +157,33 @@ public IdentityVerificationAnswersResponse submitAnswers(
     }
   }
 
+  private IdentityVerificationAnswersResponse checkAnswersAndCreateQueuedOrganization(
+      IdentityVerificationAnswersRequest requestBody, OrganizationQueueItem orgQueueItem)
+      throws IOException {
+    String orgAdminEmail = orgQueueItem.getRequestData().getEmail();
+
+    try {
+      IdentityVerificationAnswersResponse verificationResponse =
+          _experianService.submitAnswers(requestBody);
+
+      verificationResponse.setEmail(orgAdminEmail);
+
+      if (verificationResponse.isPassed()) {
+        // create and id verify the organization; create and activate admin user account
+        String activationToken = _orgQueueService.createAndActivateQueuedOrganization(orgQueueItem);
```

Comment:
We may also need to catch exceptions that can happen when creating an organization or user. AFAIK, these are just Okta `ResourceExceptions`, but there may be others I haven't considered.

Reply:
Oh, very good catch!  Yes, these should be handled here.  It should occur a lot less than before (especially the conflicting user error), but definitely will happen (I think even network errors can cause this exception).